sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Common/RuleTest.cs

```diff
@@ -196,16 +197,16 @@ public void AllRules_SonarWayTagPresenceMatchesIsEnabledByDefault()
             {
                 var isInSonarWay = analyzer.CustomTags.Contains(DiagnosticDescriptorBuilder.SonarWayTag);
 
-                if (parameterizedAnalyzers.Contains(analyzer))
-                {
-                    analyzer.IsEnabledByDefault.Should().BeFalse($"{analyzer.Id} has parameters and should be disabled by default");
-                }
-                else if (IsSecurityHotspot(analyzer))
+                if (IsSecurityHotspot(analyzer))
                 {
                     // Security hotspots are enabled by default, but they will report issues only
                     // when their ID is contained in SonarLint.xml
                     analyzer.IsEnabledByDefault.Should().BeTrue($"{analyzer.Id} should be enabled by default");
                 }
+                else if (parameterizedAnalyzers.Contains(analyzer))
+                {
+                    analyzer.IsEnabledByDefault.Should().BeFalse($"{analyzer.Id} has parameters and should be disabled by default");
```

Comment:
we don't have any SonarWay rule which is parameterized?

Reply:
I am a bit confused,  maybe a comment will help

Reply:
There are parametrized rules that are in Sonar way (S3776) , but it was decided to not enable them by default even when they are in Sonar way (see #1274). I am not sure the easoning behind this, but probably FP raised for such cases.
Here, this is the first time we have a rule beeing a hotspot and that can be parametrized. I chose to go with the hotspot behaviour in such cases (activated by default), since there is technically no FP for hotspots

Reply:
I will add a link to the issue in the code as a comment